pkg/cmd/workspace/start.go

```diff
@@ -24,6 +24,7 @@ const (
 
 var startProjectFlag string
 var allFlag bool
+var ideCodeFlag bool
```

Comment:
can't we reuse `codeFlag` from the `create.go` file?

Reply:
done